### ISSUE TITLE
How gradients are computed

### ISSUE THREAD

**Issue by AndreevP** [itayhubara/CalibTIP#2]
Hello and thank you for an interesting paper!
I have a question concerning the optimization of the quantization step size. In section D.1 of the paper, you mentioned the usage of Adam optimizer for solving the AdaQuant optimization problem, but I have failed to find a description of the methodology used for backpropagating gradients through formally nondifferentiable rounding operation. Is it just Straight Through Estimator?
Thank you for your time and consideration.

**Comment by itayhubara** [comment]
Yes - here we used STE but you can apply different techniques the main idea remains the same
